web/elm/src/Build/StepTree/StepTree.elm

```diff
@@ -774,6 +808,22 @@ viewStepWithBody model session depth { id, name, log, state, error, expanded, ve
                 , class "clearfix"
                 ]
                 ([ viewMetadata metadata
+                 , case imageCheck of
+                    Just subTree ->
+                        Html.div [ class "seq" ]
+                            [ viewTree session model subTree (depth + 1)
+                            ]
+
+                    Nothing ->
+                        Html.text ""
+                 , case imageGet of
+                    Just subTree ->
+                        Html.div [ class "seq" ]
+                            [ viewTree session model subTree (depth + 1)
+                            ]
+
+                    Nothing ->
+                        Html.text ""
```

Comment:
Not sure what the best design is here, but IMO it's a bit hard to tell that the build output for the `task` step belong to the task step itself, rather than the `get: image` (when the `get: image` header is collapsed). Sure, build logs are indented, but it's pretty subtle:

<img width="801" alt="Screen Shot 2020-10-14 at 12 08 22 PM" src="https://user-images.githubusercontent.com/26583442/96017007-72eb6c00-0e17-11eb-987e-cccb1455ab59.png">

I wonder if you could just throw in something to separate them a bit more (could just be a `<hr>`, or more padding, or *something*) - not sure if @matthewpereira has any ideas.

p.s. I like nesting the headers under `task`, esp. with the sticky headers

Reply:
Yeah good point. (btw, what happened with the task name in your screenshot?)

I've got a couple other things to fix up (added a TODO section above) so I'll see if I can think of something to make this a little more obvious in the meantime.

Reply:
> btw, what happened with the task name in your screenshot?

I was initially wondering if something in this PR broke the task header, but I realized I just had `task: ""` in the pipeline config I used

Reply:
Though interestingly it didn't trigger a config validation warning...but we probably should